homeassistant/components/smarty/sensor.py

```diff
@@ -77,6 +81,21 @@ class SmartySensorDescription(SensorEntityDescription):
         device_class=SensorDeviceClass.TIMESTAMP,
         value_fn=get_filter_days_left,
     ),
+    SmartySensorDescription(
+        key="system_state",
+        translation_key="system_state",
+        value_fn=lambda smarty: smarty.system_state,
+    ),
```

Comment:
What are the possible states?

Reply:
Instead of returning a number, we should return a string like `standby`. We can then define this in the `options` parameter and also translate this